save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/ProjectDto.kt

```diff
@@ -0,0 +1,85 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.validation.Validatable
+import com.saveourtool.save.validation.isValidEmail
+import com.saveourtool.save.validation.isValidName
+import com.saveourtool.save.validation.isValidUrl
+import kotlinx.serialization.Serializable
+
+/**
+ * Project data transfer object
+ *
+ * @property name name of a project
+ * @property organizationName name of an organization that is connected to the project
+ * @property description description of a project
+ * @property isPublic flag is project public or not
+ * @property url url of a project
+ * @property email email of a project
+ */
+@Serializable
+data class ProjectDto(
```

Comment:
`Project` should be moved to `jvmMain` and have `@Serializable` removed. Will you implement this?

Reply:
Will do it later, now there are more important tasks

Reply:
* #363